common/static/coffee/src/discussion/views/new_post_view.coffee

```diff
@@ -87,7 +87,6 @@ if Backbone?
               url: url
               type: "POST"
               dataType: 'json'
-              async: false # TODO when the rest of the stuff below is made to work properly..
```

Comment:
@jimabramson Do you know if there was ever an attempt to make this AJAX callback asynchronous? If so, do you have any recollection of what the issues were? In other words, can you think of any reason why we should not make this change?